remoteworkitem/github.go

```diff
@@ -32,6 +31,9 @@ func (f *githubIssueFetcher) listIssues(query string, opts *github.SearchOptions
 
 // Fetch tracker items from Github
 func (g *GithubTracker) Fetch() chan TrackerItemContent {
+
+	configuration := getConfigurationData()
```

Comment:
Change to pass the Configuration or preferably the required options into the object instead of recreating the configuration.